tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -210,4 +231,15 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
         return baseDevfile;
     }
 
+    async startWorkspace(workspaceId: string) {
+        Logger.debug('TestWorkspaceUtil.startWorkspace');
+
+        try {
+            await this.processRequestHandler.post(`${TestWorkspaceUtil.WORKSPACE_API_URL}/${workspaceId}/runtime?debug-workspace-start=false`);
```

Comment:
Can you please comment on what is the `debug-workspace-start=false` doing and possibly why we need it here? Thank you